main.cpp

```diff
@@ -51,85 +51,95 @@ arm.flex();                             // 3)
 arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
-//  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  10)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  1)
+//  Noun: ball
+//  action 1: The ball bounces
+ball.bounce();
+//  action 2: The ball rolls
+ball.roll();
+//  action 3: The ball spins counter clockwise
+ball.spinCounterClockwise();
+
+//  2)
+//  Noun: gate
+//  action 1: The gate opens
+gate.open();
+//  action 2: The gate closes
+gate.close();
+//  action 3: The gate locks
+gate.lock();
+
+//  3)
+//  Noun: radio
+//  action 1: The radio starts up
+radio.startUp();
+//  action 2: The radio shuts down
+radio.shutDown();
+//  action 3: The radio plays sound
+radio.playSound();
+
+//  4)
+//  Noun: person
+//  action 1: This person hums
+person.hum();
+//  action 2: This person laughs
+person.laugh();
+//  action 3: This person walks backwards
+person.walkBackwards();
+
+//  5)
+//  Noun: airplane
+//  action 1: The airplane ascends
+airplane.ascend();
+//  action 2: The airplane descends
+airplane.descend();
+//  action 3: The airplane deploys its landing gear
+airplane.deployLandingGear();
+
+//  6)
+//  Noun: scene
+//  action 1: The scene starts
+scene.start();
+//  action 2: The scene pauses 
+scene.pause();
+//  action 3: The scene stops
+scene.stop();
+
+//  7)
+//  Noun: cat
+//  action 1: The cat meows
+cat.meow();
+//  action 2: The cat scratches
+cat.scratch();
+//  action 3: The cat eats
+cat.eat();
+
+//  8)
+//  Noun: phone
+//  action 1: The phone rings
+phone.ring();
+//  action 2: The phone plays a dial tone
+phone.playDialTone();
+//  action 3: The phone disconnects
+phone.disconnect();
```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.